src/fromPairsMapNonNil.js

```diff
@@ -0,0 +1,15 @@
+var _ = require('lodash')
```

Comment:
Let's use `const` here

```suggestion
const _ = require('lodash')
```